docs/platforms/react-native/user-feedback/configuration/index.mdx

```diff
@@ -14,6 +14,9 @@ import * as Sentry from "@sentry/react-native";
 
 Sentry.showFeedbackWidget();
 ```
```

Comment:
I would even add it to the code example/

````suggestion
Sentry.showFeedbackWidget();

Sentry.wrap(RootComponent);
```
````

Reply:
Good idea @krystofwoldrich 👍 
I've also added it to the other example and added links to the related section of the installation page with d3352e755195b3eaa9f13dd99c973b9dd6aa6781